README.rst

```diff
@@ -62,65 +176,231 @@ A simple example to get you up and running:
   print("Results:")
   print(result)
 
-Example output:
+You should see the following output printed by Python:
```

Comment:
Nit - the result `m` is random and will likely differ in user's run

```suggestion
You should see a similar output printed by Python:
```

Reply:
Will fix in the md version.